docs/sources/next/set-up/configure-your-code-editor.md

```diff
@@ -36,4 +37,10 @@ $ npm install --save-dev @types/k6
 - [Visual Studio Code - k6 Extension](https://marketplace.visualstudio.com/items?itemName=k6.k6)
 - [IntelliJ IDEA - k6 Plugin](https://plugins.jetbrains.com/plugin/16141-k6)
 - [TypeScript Editor Support](https://github.com/Microsoft/TypeScript/wiki/TypeScript-Editor-Support)
```

Comment:
I'm wondering if we still need this section? Are these extensions still being maintained?